sdl2-sys/sdl_bindings.rs

```diff
@@ -24387,7 +24387,7 @@ extern "C" {
         dstrect: *const SDL_Rect,
         angle: f64,
         center: *const SDL_Point,
-        flip: SDL_RendererFlip,
+        flip: u32,
```

Comment:
is this a good way to fix this issue? this file is (partly?) generated, but the original function signature did not allow `SDL_FLIP_HORIZONTAL | SDL_FLIP_VERTICAL`

Reply:
This file is always fully generated and never modified manually (plus this breaks past code using sdl2-sys raw), so please revert that and adapt.